src/lib/client.ts

```diff
@@ -169,6 +169,13 @@ export class Client {
     await this.publicApi.isReady
   }
 
+   /* Disconnects from PUBLIC-rpc node - Example: farm-rpc.subspace.network */
+  public async disconnectPublicApi(): Promise<void> {
+    if (this.publicApi.isConnected) {
+      await this.publicApi.disconnect()
+    }
```

Comment:
I'm not sure check for connected is necessary. It may be racy with this check in place.

Reply:
ok, i will change it, no problem :)